### DISCUSSION TITLE
Redundant special methods

### DISCUSSION THREAD

**Discussion by stasoid** [litehtml/litehtml#257] [Q&A] [answered]
I again encountered some classes (css_position, background) with copy constructor/copy assignment operator that behave exactly as implicitly generated ones. I don't understand the purpose of those. Can I remove them?

Same for default constructors and destructors. I don't understand why `~background() = default;` or `css_borders() = default;` are needed. Can I remove them?

**Comment by tordex** [comment, marked as answer]
> Can I remove them?

Yes, it is safe to remove these.
